starter/script.js

```diff
@@ -1 +1,119 @@
-// put code here!
+console.log("JS connected")
+
+//thoughts about this: should I use jquery or really stick with vanilla?
+// could use a combo.
+//whats the learning objective here?
+
+
+// Rules and contents
+// Any Path that the user goes down must ask the user at least 3 questions
+// must have a total of seven destinations
+// Ending Zones:
+// 1) Niflheim: The World of Fog and Mist
+// 2) Muspelheim: The land of fire
+// 3) Asgard: Home of the gods
+// 4) Midgard: Home of the humans (earth)
+// 5) Jotunheim: Home of the Giants
+// 6) Vanaheim: Home of the Vanir
+// 7) Alfheim: Home of the light elves
+// 8) Svartalfheim: Home of the Dwarves
+// 9) Helheim: Home of the dishonorable dead
+
+// one potential ending:
+// “For you I’ve become the most subtle creature of them all: a civilized evil, genteel and seductive. An evil you endure because you need it’s 
+//service- even though that very endurance plucks loose the underpinnings of your morality. An evil that causes you to question the definition of your identity, that blurs the line between dark and light until you’re no longer certain 
+//which is which, or how the two are different- That’s what you fear most, priest.”
+
+
+// -------------------------------------
+
+ //the script below will be used to input text back onto 
+ //the page, what we wana do is apend the info with next in the 
+ //chain but keep the previous directions down the tree will 
+ //need to "some text to enter"/ replace it with the return value of a 
+ // function that selects the next possible path. -->
+
+// ------function below for above-----
+// 
+
+
+
+function insertText () {
```

Comment:
I imagine this function needs an argument, if the text that is to be inserted varies which each user action.